src/themes/ThemeProvider.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react'
+
+import underscore from '../utils/underscore'
+
+type ThemeProviderProps = {
+    theme: string
+}
+
+/**
+ * Exposes a theme to all Reactist components.
+ *
+ * Usage:
+ *
+ *  <ThemeProvider theme="twist-light" />
+ *
+ * somewhere in your React component tree.
+ *
+ * Note: Only one instance of ThemeProvider must be rendered. To override themes for a single component,
+ * use the `theme` prop for that component.
+ */
+export default function ThemeProvider({ theme }: ThemeProviderProps) {
```

Comment:
Probably this is enough for the initial use case but looking both at Todoist and Twist I think we want to turn this into a real context provider that exposes the theme and a function to change the theme. Should we add this immediately or keep it simple for as long as possible?

Reply:
Looking at the Twist PR I think we should keep it simple :D